docs/Examples.md

```diff
@@ -85,3 +85,63 @@ afs_models.get_latest_model_info(model_repository_name='model.h5')
 	'created_at': '2018-09-11 10:15:54'
 }
 ```
+
+
+### upload_model (big model)
+
+How to upload a big model (300MB-1GB) file on notebook. 
+
+**Code**
+
+```
+from afs import models
+
+# Write a big file as 301 MB model file.
+f = open('big_model.h5', "wb")
+f.seek((301 * 1024 * 1024 + 1) - 1)
+f.write(b"\0")
+f.close()
+
+# User-define evaluation result
+extra_evaluation = {
+    'AUC': 1.0
+}
+
+# User-define Tags 
+tags = {'machine': 'machine01'}
+
+# Model object
+afs_models = models()
+afs_models.set_blob_credential(
+	blob_endpoint="http://x.x.x.x:x", encode_blob_accessKey="ENCODE_BLOB_ACCESSKEY", encode_blob_secretKey="ENCODE_BLOB_SECRETKEY"
+	)
```

Comment:
Is there any documentation to instruct users how to get encoded blob keys? And, in the task, is it possible to set different credential or left it blank to make task provide credential automatically?

Reply:
1. I will add the encode document on doc.
2. It might be implemented in `get_env` to fetch the variable if the blob credential existed, but it has not implement in server. This version user need to set in hard code.